src/components/Accordion/AccordionItem.tsx

```diff
@@ -0,0 +1,69 @@
+import { px } from '../../utils';
+import { getCommonProps } from '../../utils';
+import { AccordionItemType } from './types';
+import plusIcon from '../../assets/plus.svg';
+import minusIcon from '../../assets/minus.svg';
+import lockIcon from '../../assets/lock.svg';
+import lockBlueIcon from '../../assets/lockBlue.svg';
+import { useState } from 'react';
+import classNames from 'classnames';
+
+/**
+ * ## Overview
+ *
+ * A single Accordion Item in a list
+ */
+
+const AccordionItem = ({ ...props }: AccordionItemType) => {
+  getCommonProps(props, 'AccordionItem');
+  const { locked, lockedVariation, lockedContent, variation, label, content, isLastItem, id } = props;
+  const [contentState, setContentState] = useState('collapsed');
```

Comment:
Would be better to be a boolean?

```suggestion
  const [isCollapsed, setIsCollapsed] = useState('false');
```